packages/mdc-select/foundation.js

```diff
@@ -190,8 +215,21 @@ export default class MDCSelectFoundation extends MDCFoundation {
     const focusIndex = this.selectedIndex_ < 0 ? 0 : this.selectedIndex_;
 
     this.setMenuStylesForOpenAtIndex_(focusIndex);
+    this.adapter_.addClassToLabel(cssClasses.LABEL_FLOAT_ABOVE);
+    this.adapter_.addClassToBottomLine(cssClasses.BOTTOM_LINE_ACTIVE);
     this.adapter_.addClass(OPEN);
     this.adapter_.openMenu(focusIndex);
+    this.isFocused_ = true;
+  }
+
+  setBottomLineOrigin_(evt) {
+    const targetClientRect = evt.target.getBoundingClientRect();
```

Comment:
If the client decides to put child elements inside of the option, wouldn't `evt.target` potentially point to the wrong element? E.g.:

```html
<li class="mdc-list-item" role="option" tabindex="0">
  <div class="foo">
    <i class="material-icons">date_range</i>
    Date Range
  </div>
</li>
```

Reply:
I...actually don't think there would be a consequence to this. The method is setting the transform origin in the x direction for the bottom line to be the location of the pointer along the x-axis. This would (I think) have the same result no matter what children are actioned on. I will test this, but I suspect we're fine as is with this.